contracts/v2/sovereignChains/GlobalExitRootManagerL2SovereignChain.sol

```diff
@@ -0,0 +1,112 @@
+// SPDX-License-Identifier: AGPL-3.0
+
+pragma solidity 0.8.20;
```

Comment:
Solc compiler version 0.8.20 switches the default target EVM version to Shanghai, which means that the generated bytecode will include PUSH0 opcodes. Be sure to select the appropriate EVM version in case you intend to deploy on a chain other than mainnet like L2 chains that may not support PUSH0, otherwise deployment of your contracts will fail.